### ISSUE TITLE
add SVG support

### ISSUE THREAD

**Issue by MichelDiz** [TheSavior/react-native-mask-loader#6]
SVG is much better, PNG or others get distorted. Expo has SVG support by the way.

**Comment by elicwhite** [comment]
Yep. Svg is definitely the right choice since the image scales up and becomes blurry. I’d accept a PR!

**Comment by MUZAFFAR-IQBAL-NARSUN** [comment]
@TheSavior have you add svg support?. If don't please add i need it. Thanks
